src/BeamAdapter/component/mapping/AdaptiveBeamMapping.cpp

```diff
@@ -195,6 +195,22 @@ void AdaptiveBeamMapping<Rigid3Types, Rigid3Types >::computeJacobianOnPoint(unsi
     dmsg_info()<<" ********** TEST J-Jt(transposed): ********** \n"<<Test;
 }
 
+
+template <>
+SOFA_BEAMADAPTER_API int AdaptiveBeamMapping<Rigid3Types, Rigid3Types >::addPoint (const Coord& point, int indexFrom)
+{
+    SOFA_UNUSED(indexFrom);
+
+    int nbPoints = d_points.getValue().size();
+    Vec3 coord = point.getCenter();
+
+    auto points = sofa::helper::getWriteOnlyAccessor(d_points);
+    points.push_back(coord);
```

Comment:
Since `coord` is a copy, I wonder if the compiler managed to move it in the `push_back` or copied it.